core/src/components/list-header/list-header.ionic.scss

```diff
@@ -0,0 +1,55 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@import "./list-header";
+@import "../item/item.ionic.vars";
+
+:host {
+  --background: transparent;
+
+  min-height: 58px;
```

Comment:
Where is this coming from?

Reply:
It comes from Figma when inspecting the list header.
![Screenshot 2024-06-25 at 9 03 15 AM](https://github.com/ionic-team/ionic-framework/assets/13530427/fa78208e-a677-4a47-9a3f-3ecd2d6b2698)